recipe/meta.yaml

```diff
@@ -20,6 +20,9 @@ requirements:
     - pkg-config
     - make
     - cmake
+    - libgit2
```

Comment:
This needs to be in host

Reply:
I had tried that but got a different error. Let me update it again because somehow the logs were deleted.